src/pages/scientificServices/NavPaths.tsx

```diff
@@ -0,0 +1,11 @@
+import { CliAuth } from 'src/pages/scientificServices/cli-auth/CliAuth';
+
+export const navPaths = [
+  {
+    name: 'cli-auth',
+    path: '/pipelines/cli-auth',
```

Comment:
to be sure I understand, based on this we expect the full path in prod to be `https://app.terra.bio/#pipelines/cli-auth?code=testcode1`, is that right? 

is `pipelines` going to be used elsewhere for teaspoons/imputation service UI pages?